src/componentP/Profile.js

```diff
@@ -7,10 +7,16 @@ import {
 
 const MyProfile = () => {
   const rockets = useSelector((state) => state.rock);
+  const missions = useSelector((state) => state.missions);
   return (
     <Profile>
-      <div style={{ width: '50%' }}>
-        My mission
+      <div className="joined-missions">
+        <h3>My Missions</h3>
+        {missions.map((record) => (record.joined ? (
+          <div className="joined-mission-item">
+            <h4 className="joined-mission-title">{record.name}</h4>
+          </div>
+        ) : null))}
```

Comment:
On this part you miss to give a `key` value to the items that we are showing on the UI..

however, I'll add that so we can ask for review.

Remember that without a `key` to identify that our items are unique we will receive an error in the console